netlify.toml

```diff
@@ -2,20 +2,20 @@
   from = "https://doc.silverstripe.org/*"
   to = "https://docs.silverstripe.org/:splat"
   force = true
-  status = 301
+  status = 302
```

Comment:
What's the rational for making this a temporary redirect?

Reply:
So it doesn't get cached e.g. browser cache.

This should have been a 302 to being with. 301's should only be used for things never intended to be changed again in the future. This will changed again in the future when we change to CMS 6.

Reply:
I think we have different understanding of the best practice here. In my mind, a "temporary" redirect is something that will last a few hours or a few days. e.g.: Something is broken on the site and we will send you somewhere else while we work on fixing it. 

Something that will last for two years, is functionally permanent.

If we set it to a 302, won't Google want to keep the version 4 of the doc as the "default" one in search results?

Reply:
It's more an issue that 301 redirects can get cached at various points where traffic is served, including in browser caches. This means that browsers that previously visited a url will still redirect to the old old location.  302 doesn't get cached so it doesn't have this problem.

I'd expect google is smart enough to handle changing a 301 to a different location, I'd expect it's a fairly common occurrence that people change decide that a 301 should point somewhere else after some time

We also use 302 on api.silverstripe.org for the default redirect - https://github.com/silverstripe/api.silverstripe.org/blob/master/.htaccess#L49